### PR TITLE
website deployed to firebase

### PR DESCRIPTION
**Pull request by DreRandaci**
https://angular-contacts-ff671.firebaseapp.com

### CODE REVIEW

**Comment by DreRandaci** [comment]
https://angular-contacts-ff671.firebaseapp.com

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <script type="text/javascript" src="lib/node_modules/angular/angular.min.js"></script>
+    <script type="text/javascript" src="lib/node_modules/angular-animate/angular-animate.min.js"></script>
+    <script type="text/javascript" src="lib/node_modules/angular-route/angular-route.min.js"></script>
+    <script type="text/javascript" src="lib/node_modules/angular-sanitize/angular-sanitize.min.js"></script>
+    
+    <script type="text/javascript" src="javascripts/app.js"></script>
+    <script type="text/javascript" src="javascripts/AppConstants.js"></script>
+    <script type="text/javascript" src="javascripts/AppConfig.js"></script>
+
+    <!-- <script type="text/javascript" src="javascripts/services/AuthService.js"></script> -->
```
Comment (morecallan) [inline index.html]: Remove this for this PR.

File: javascripts/AppConfig.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+app.config(function($routeProvider){
+    $routeProvider
+    .when( "/login", {
+        templateUrl: 'partials/login.html',
+        controller: 'LoginCtrl'
+    })
+    .when( "/contacts-favorites", {
```
Comment (morecallan) [inline javascripts/AppConfig.js]: These should be formatted `contacts/favorites/`. (Same with the next 2 routes).

File: lib/Gruntfile.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          dev: {
+            files: [{
+              expand: true,
+              cwd: "../",
+              src: [
+                "index.html",
+                "partials/contacts-favorites.html",
+                "partials/navbar.html",
+                "partials/contacts-new.html",
+                "partials/contacts-view.html",
```
Comment (morecallan) [inline lib/Gruntfile.js]: You might want to include every html in your partials (and every JS in your javascripts) or you will really end up hating angular 😡

**Comment by rousell** [comment]
@DreRandaci 👍 looks good! merging in

**Pull request merged**